tests/com/haskforce/highlighting/annotation/external/GhcUtilTest.java

```diff
@@ -0,0 +1,53 @@
+package com.haskforce.highlighting.annotation.external;
+
+import com.intellij.openapi.editor.LogicalPosition;
+import com.intellij.openapi.editor.VisualPosition;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by kasper on 12/24/14.
```

Comment:
I mainly added javadoc on class level, not on method level or so. If you think more is needed, let me know.